tasks/src/question-answering/about.md

```diff
@@ -1,2 +1,32 @@
-Question answering is a natural language understanding task. Question answering models allow users to search for an answer in a document. There are two question answering types, one is open-domain question answering, and the other one is closed domain question answering. Open-domain question answering models are not restricted to a specific domain, meanwhile closed-domain questions answering models are restricted to a specific domain (e.g. legal, medical documents). According to taking context information, there are two types of question answering problems. Open book question answering models take a context with a question where answer is extracted by the context. Closed book question answering models takes no context and are expected to know answer to question, and generate the answer rather than extracting. There are various question answering variants based on the input and output they return, being, extractive question answering (that is solved with BERT-like models), generative question answering (that is solved with models like BART and T5) and Table Question Answering (solved with TAPAS model).
+## Use Cases
+
+You can use question answering models with knowledge base as contexts and user inputs as questions to automate frequently asked questions. You need documents containing your company’s informations and a question answering model either from the Hub or trained by you. Questions coming from your customers can be inferred from those documents. If you’d like to save inference time, you can first use passage ranking models to see which document might contain the answer to the question and iterate over that document with the question answering model instead.
+
+## Task Variants
+There are two question answering types, one is open-domain question answering, and the other one is closed domain question answering. Open-domain question answering models are not restricted to a specific domain, meanwhile closed-domain questions answering models are restricted to a specific domain (e.g. legal, medical documents). According to taking context information, there are two types of question answering problems. Open book question answering models take a context with a question where answer is extracted by the context. Closed book question answering models takes no context and are expected to know answer to question, and generate the answer rather than extracting. There are various question answering variants based on the input and output they return, being, extractive question answering (that is solved with BERT-like models), generative question answering (that is solved with models like BART and T5) and Table Question Answering (solved with TAPAS model).
```

Comment:
This paragraph might be a bit too long, I would suggest to split it a bit or use numbered lists when it makes sense.